pkg/cluster/install.go

```diff
@@ -93,6 +94,18 @@ func (m *manager) adminUpdate() []steps.Step {
 		)
 	}
 
+	if isRenewCerts {
+		toRun = append(toRun,
+			steps.Action(m.fixMCSCert),
+			steps.Action(m.configureAPIServerCertificate),
+			steps.Action(m.configureIngressCertificate),
+			steps.Action(m.initializeOperatorDeployer), // depends on kube clients
+			steps.Action(m.renewMDSDCertificate),
+		)
+		// return early, because we only want to run the certificate actions using the existing operator
+		return toRun
```

Comment:
If we use the above pattern, we don't really need to return early.

Reply:
Will remove the step `return toRun` as its not needed in this case.